linera-rpc/tests/staged/formats.yaml

```diff
@@ -1,931 +0,0 @@
----
```

Comment:
Was this accidentally deleted?

Reply:
Yes :(  I got annoyed at forgetting to update this so I set up a commit hook — but then I killed it in the middle and it got deleted :sweat_smile: Thanks for the catch!